manual/tests/Makefile

```diff
@@ -2,19 +2,26 @@ TOPDIR=$(abspath ../..)
 include $(TOPDIR)/Makefile.tools
 MANUAL=$(TOPDIR)/manual/manual
 
-.PHONY: all
 all: check-cross-references check-stdlib
 
+
 cross-reference-checker: cross_reference_checker.ml
-	$(OCAMLC) $(TOPDIR)/compilerlibs/ocamlcommon.cma -I $(TOPDIR)/parsing \
-         -I $(TOPDIR)/driver \
-        cross_reference_checker.ml -o cross-reference-checker
+	$(OCAMLC) $(TOPDIR)/compilerlibs/ocamlcommon.cma \
+	  -I $(TOPDIR)/parsing -I $(TOPDIR)/driver \
+	  $< -o $@
 
+.PHONY: check-cross-references
 check-cross-references: cross-reference-checker
 	$(OCAMLRUN) ./cross-reference-checker \
```

Comment:
While we are at it, this `$(OCAMLRUN)` invocation should probably be `$(SET_LD_PATH) $(OCAMLRUN)` as the others; see 482e8a8b52ff9fb8e8f8aecb17264f314b9a37f0.